interfaces/builtin/kvm.go

```diff
@@ -38,8 +48,60 @@ const kvmConnectedPlugAppArmor = `
 
 var kvmConnectedPlugUDev = []string{`KERNEL=="kvm"`}
 
+type kvmInterface struct {
+	commonInterface
+}
+
+var procCpuinfo = "/proc/cpuinfo"
+
+func getCpuFlags() (flags string, err error) {
```

Comment:
How about returning a list of strings here with the flags? Something like: `func getCpuFlags() (flags []string, err error` ? We have strutil.ListContains() (and the same for sorted lists) as a helper so that you can easily test if a string is in the list. Or even a map[string]bool  (or interface{}) ? Of course with the list approach once the flags are identified an additional `flags = strings.Fields(flagsString)` will be required but I think thats not too bad.

Reply:
Done